### PR TITLE
integrations: Add screenshots and notes for solano

### PR DESCRIPTION
**Pull request by anirudhjain75**

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@anirudhjain75, it looks like you've already signed the Dropbox CLA. Thanks!

File: webhook-integrations/solano/notes.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Solano Lab Integration
+
+Solano CI is a hosted testing and continuous-delivery service that
+speeds up your development process. Solano CI:
+
+* Integrates with your existing test suites and build processes
+
+* Supports a wide array of languages and tools
+
+* Fits into your git or mercurial workflow
```
Comment (umairwaheed) [inline webhook-integrations/solano/notes.md]: Git or Mercurial.

File: webhook-integrations/solano/notes.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+* Automatically runs your build steps and tests in parallel
+so they finish faster
+
+For adding slack Integration follow the steps given below:
```
Comment (umairwaheed) [inline webhook-integrations/solano/notes.md]: Slack. Comma after 'Integration'. Maybe 'To add Slack Integration'?

File: webhook-integrations/solano/notes.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+* Make an account in [solano](https://support.solanolabs.com/hc/en-us).
+
+* Click on **Organisation Settings**.
+
+* Choose the **chat notification** in the **Organisation Settings** page.
+
+* Scroll down to Slack and enter your slack account details in the Slack option
```
Comment (umairwaheed) [inline webhook-integrations/solano/notes.md]: Slack account.

**Comment by umairwaheed** [comment]
@anirudhjain75, looks good. Few comments added.

**Comment by anirudhjain75** [comment]
@umairwaheed Made the changes you requested, Thanks

**Comment by TomaszKolek** [comment]
@anirudhjain75 Why didn't you add any Slack messages screenshots? They are required in task description. Could you update it?

**Comment by showell** [comment]
merged, thanks, @anirudhjain75 !

**Pull request merged**